core/common/src/rpc/core/RpcMarshaling.ts

```diff
@@ -115,7 +115,7 @@ class WireFormat {
   }
 
   private static marshalBinary(value: any): any {
-    if (value instanceof Uint8Array || Buffer.isBuffer(value)) {
```

Comment:
Why isn't this available in esbuild? Were we somehow polyfilling it with our webpack build?

Reply:
Appears so. Likely from babel / react-app-polyfill

This module is not available in the browser. The magic that we occasionally take for granted is going to be one of our biggest paint points when attempting to move towards the latest generation of es module based build tools. I'm sure there are/will be plugins to help the transition.